src/frontend/components/app/filter-form.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Button, DEFAULT_DRAWER_WIDTH, H3, Icon } from '@adminjs/design-system'
```

Comment:
`'Icon' is defined but never used`

Reply:
I removed this component because it was only for local development and maybe can be introduced in the future to allow users to show filters not as a drawer but more as a side panel of list

![image](https://user-images.githubusercontent.com/85543813/233043173-8fb68ef7-cb97-4f45-81f1-2fce2a418779.png)